xoa_driver/functions/anlt.py

```diff
@@ -213,8 +278,10 @@ async def an_status(port: GenericAnyPort) -> Dict[str, Any]:
     :rtype: typing.Dict[str, Any]
     """
     conn, mid, pid = port._conn, port.kind.module_id, port.kind.port_id
```

Comment:
Be aware `port._conn` is marked as a  private for a reason, because it can be renamed or changed its location under HLI and your code will stop working